assets/js/index.js

```diff
@@ -23,17 +26,25 @@ async function postTodo(todoText) {
   }
 }
 
-addTaskIcon.addEventListener("click", function () {
-  postTodo(addTaskInput.value);
+addTaskIcon.addEventListener("click", async function () {
+  await postTodo(addTaskInput.value);
   clearAddTodoInput();
   clearErrorMessage();
+  const tasks = await getTasks();
+  showTasks(tasks);
+  filterItemAll.innerText = tasks.length;
+  filterItemUncompleted.innerText = tasks.length;
 });
 
-addTaskInput.addEventListener("keydown", (e) => {
+addTaskInput.addEventListener("keydown", async (e) => {
   if (e.key === "Enter") {
-    postTodo(addTaskInput.value);
+    await postTodo(addTaskInput.value);
     clearAddTodoInput();
     clearErrorMessage();
+    const tasks = await getTasks();
+    showTasks(tasks);
+    filterItemAll.innerText = tasks.length;
+    filterItemUncompleted.innerText = tasks.length;
```

Comment:
Move these to a stand alone function and just call the function here.

Reply:
Done.